crates/sp-domains-fraud-proof/src/runtime_interface.rs

```diff
@@ -0,0 +1,20 @@
+#[cfg(feature = "std")]
+use crate::FraudProofExtension;
+use crate::InvalidDomainExtrinsicRootInfo;
+use sp_core::H256;
+#[cfg(feature = "std")]
+use sp_externalities::ExternalitiesExt;
+use sp_runtime_interface::runtime_interface;
+
+/// Domain fraud proof related runtime interface
+#[runtime_interface]
+pub trait FraudProofRuntimeInterface {
+    fn get_invalid_domain_extrinsic_root_info(
```

Comment:
There should be description for such host functions. Also the name is odd, is it really about invalid domain extrinsic root or is it about any, it just happens to be used for invalid extrinsics by the runtime?